lib/linter-glsl.js

```diff
@@ -183,6 +207,11 @@ export default {
       default: false,
       order: 2,
     },
+    useGlslify: {
```

Comment:
This probably should have a `description` on here describing what this is actually doing.